src/components/AddTodo/AddTodo.jsx

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import './AddTodo.css';
+import { UserType } from '../../types';
+
+export class AddTodo extends React.Component {
+  state = {
+    users: this.props.users,
+    selectedUserId: '',
+    title: '',
+    requireUserSelection: false,
+    requireTitle: false,
```

Comment:
`require...` название функции
запроситьТайтл
запроситьВыборЮзера
```suggestion
    requiredUserSelection: false,
    requiredTitle: false,
```